pinot-plugins/pinot-file-system/pinot-adls/src/main/java/org/apache/pinot/plugin/filesystem/ADLSGen2PinotFS.java

```diff
@@ -96,6 +99,12 @@
   // However, there's some overhead in computing hash. (Adds roughly 3 seconds for 1GB file)
   private boolean _enableChecksum;
 
+  public ADLSGen2PinotFS(DataLakeFileSystemClient fileSystemClient,
```

Comment:
Please add javadoc on all public methods/apis.

Reply:
There is no more details to add here, its a constructor and the signature is self documenting. Also looking at the other classes, doesn't seem to be a convention for the code base.
But I'm guessing you meant for the other methods that are not in this change. So added docs for all of them.